MonMaraicher/Model/FarmerPlace.swift

```diff
@@ -13,27 +13,97 @@ struct FarmerPlace: Identifiable, Hashable {
+                         zip: 34790,
+                         city: "grabels")
+    )
+    private static let fred = Location(
+        latitude: 43.66331635156326,
+        longitude: 3.9341394263737,
+        address: Address(streetNumber: nil,
+                         streetName: "D65",
+                         zip: 34920,
+                         city: "le crès")
+    )
+    private static let maraicher = Location(
+        latitude: 43.594984821459704,
+        longitude: 3.900391797826057,
+        address: Address(streetNumber: nil,
+                         streetName: "rue de la rauze",
+                         zip: 34070,
+                         city: "montpellier")
+    )
+    private static let chot = Location(
+        latitude: 43.578405305924235,
+        longitude: 3.8071850039225246,
+        address: Address(streetNumber: nil,
+                         streetName: "D5E3",
+                         zip: 34880,
+                         city: "lavérune")
+    )
 
-    private static let applePark1 = Location(latitude: 37.33771487954408, longitude: -122.01871382213936, city: "Montpellier")
-    private static let applePark2 = Location(latitude: 37.34320749025098, longitude: -122.00085869214455, city: "Grabels")
-    private static let applePark3 = Location(latitude: 37.320788995175214, longitude: -122.02416356085007, city: "Le Crès")
-    private static let applePark4 = Location(latitude: 37.32238963337884, longitude: -121.98463768108974, city: "Montpellier")
-    private static let applePark5 = Location(latitude: 37.34986124022765, longitude: -122.01214762927336, city: "Lavérune")
+    private static let applePark1 = Location(
+        latitude: 37.33771487954408,
+        longitude: -122.01871382213936,
+        address: Address(streetNumber: 775,
+                         streetName: "avenue du maréchal leclerc",
+                         zip: 34000,
+                         city: "montpellier")
+    )
+    private static let applePark2 = Location(
+        latitude: 37.34320749025098,
+        longitude: -122.00085869214455,
+        address: Address(streetNumber: nil,
+                         streetName: "chemin du redonnel",
+                         zip: 34790,
+                         city: "grabels")
+    )
+    private static let applePark3 = Location(
+        latitude: 37.320788995175214,
+        longitude: -122.02416356085007,
+        address: Address(streetNumber: nil,
+                         streetName: "D65",
+                         zip: 34920,
+                         city: "le crès")
+    )
+    private static let applePark4 = Location(
+        latitude: 37.32238963337884,
+        longitude: -121.98463768108974,
+        address: Address(streetNumber: 139,
+                         streetName: "rue de la rauze",
+                         zip: 34070,
+                         city: "montpellier")
+    )
+    private static let applePark5 = Location(
```

Comment:
You could even write all these data inside a json file you load at application start. When you will have a backend you will need to do that

Reply:
Let is like that for now it's only suggestion for the future ;)